implementations/app-python/.nvmrc

```diff
@@ -0,0 +1 @@
+v16
```

Comment:
We've been using `v18.15`, `v16` does not have good wasm support, so we want to use a version >16.